terraform/workspaces/infra/cloudtrail/migrations.tf

```
@@ -0,0 +1,15 @@
+# v2.0.0 -> v2.1.0: conditionally create cloudtrail bucket depending on `disable_cloudtrail` argument
+moved {
+  from = aws_s3_bucket.cloudtrail
+  to   = aws_s3_bucket.cloudtrail[0]
+}
+
+moved {
+  from = aws_s3_bucket_policy.cloudtrail
+  to   = aws_s3_bucket_policy.aws_s3_bucket_policy[0]
```

```suggestion
  to   = aws_s3_bucket_policy. cloudtrail[0]
```
I think this is what it should be?